.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    branches: [ prod ]
+
+name: Deploy VEX API on AWS ECR
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
```

Comment:
For security reasons, can we use SHA1 hashes instead of tags. See https://github.com/proximacapital/typescript-template/ for format but essentially just `actions/checkout@123abc`